populate/src/main/resources/seed/iam/permissions.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "name": "study_settings_edit",
+    "displayName": "Study settings",
+    "description": "Configure study settings password protection and enrollment"
+  },
+  {
+    "name": "survey_edit",
+    "displayName": "Survey form management",
+    "description": "View, edit and configure surveys"
+  },
+  {
+    "name": "consent_form_edit",
+    "displayName": "Consent form management",
+    "description": "View, edit and configure consent forms"
+  },
+  {
+    "name": "site_content_edit",
+    "displayName": "Site content management",
+    "description": "View, edit and configure the portal website"
+  },
+  {
+    "name": "team_roles_edit",
+    "displayName": "Team role management",
+    "description": "View and edit roles assigned to team members"
+  },
```

Comment:
These roles are right now unenforced, right? Is there a followup PR to enforce these?

Reply:
you're right- so far these existing permissions have only been enforced in the frontend for hiding/disabling functionality, but there was nothing on the backend actually checking for these permissions. I'll add a ticket to backfill enforcement in the backend